src/app.js

```diff
@@ -45,13 +46,18 @@ passport.use(
 // Open Routes
 app.use('/.well-known', wellKnownRouter);
 app.use('/public', publicRouter);
+
+// Routes for collections
+Object.values(collections).forEach(collectionName => {
+  app.use(`/${collectionName}`, genericController(collectionName));
```

Comment:
This misses out on the authorization piece
```suggestion
  app.use(`/${collectionName}`, backendAuthorization, genericController(collectionName));
```

Reply:
I told him leave it open for easier testing since backendAuthorization isn't the right auth for this anyway - that one is for other apps to connect to this via SMART backend services auth. 
For these we're going to want to use an "admin user" auth so that logged in users can access these, so hopefully we get one of those for free with your upcoming login changes. (And if you don't need one for that task we'll make it a new task)

Reply:
Authorization will be added in the future so I'll just add a comment so we don't forget.